staging/src/k8s.io/kubectl/pkg/cmd/create/create_job.go

```diff
@@ -277,7 +307,7 @@ func (o *CreateJobOptions) createJobFromCronJob(cronJob *batchv1beta1.CronJob) *
 			OwnerReferences: []metav1.OwnerReference{
 				{
 					APIVersion: batchv1beta1.SchemeGroupVersion.String(),
-					Kind:       cronJob.Kind,
+					Kind:       "CronJob",
```

Comment:
why the change?

Reply:
We're not doing conversions like we did before and this field is missing from cronjob read from the server. It has to be set explicitly like in all other create commands.